docker-compose.yml

```diff
@@ -71,7 +67,9 @@ services:
     build:
       context: ./apps/analyzer
       dockerfile: Dockerfile
-    command: ['/app/.venv/bin/python3', 'main.py', '--host', '0.0.0.0']
+    #command: ['/app/.venv/bin/python3', 'main.py', '--host', '0.0.0.0']
```

Comment:
if the commented out lines are no longer needed, I would remove them to tidy up the file